charts/jenkins/values.yaml

```diff
@@ -72,6 +72,7 @@ jenkins:
       - matrix-auth
       - matrix-project
       - pipeline-build-step
+      - pipeline-github
```

Comment:
as part of the chart PRs, we'd like to add a comment with the `helmfile diff` back onto the PR, this will help us see what changes are going to be applied.

Reply:
another option would be to publish it to a github check?